authors/apps/profiles/tests/test_follow.py

```diff
@@ -145,3 +145,59 @@ def test_get_all_followers(self):
 
         res = self.client.get('/api/profiles/ironman/followers/')
         self.assertEqual(res.status_code, status.HTTP_200_OK)
+
+    def test_user_not_found_follow(self):
```

Comment:
Hello @Amoswachira ,
These tests are not part of the scope of this pull request, so I don't think it should be included

Reply:
@Ogutu-Brian the reason those are there is that I forgot to checkout from develop branch and  i checkout from my fix profiles branch

Reply:
Can you update this pull request and include only the changes related to your task.

Reply:
@oluwagbenga-joloko okay let me do that